script/vsts/nightly-release.yml

```diff
@@ -1,15 +1,15 @@
 jobs:
   - job: GetReleaseVersion
     pool:
-      vmImage: 'windows-latest'
+      vmImage: 'ubuntu-latest'
     steps:
       # This has to be done separately because VSTS inexplicably
       # exits the script block after `npm install` completes.
       - script: |
-          cd script\vsts
-          npm install
+          cd script/vsts
+          npm install request-promise-native yargs
         displayName: npm install
-      - script: node script\vsts\get-release-version.js --nightly
+      - script: node script/vsts/get-release-version.js --nightly
```

Comment:
Could you make these a template?

Reply:
Separate PR strikes me as the right approach. Easier to look back at the history and see what happened.

Will do it right away though after/if this gets merged.